### PR TITLE
User experience updates

### PR DESCRIPTION
**Pull request by Aimsucks**
- Updated all of the tab names to be more clear on what they do
- Updated settings collapsible menu names to be more clear
- Changed wording and labels on some settings
- Added tooltips for some settings
- Updated autorotation window UI elements to be a bit more clear (see screenshot and [0cb6560](https://github.com/awgil/ffxiv_bossmod/pull/465/commits/0cb65604244e7640d264420e0a99a24d7a045951))
image
- Added an "about" tab ([80c7ea8](https://github.com/awgil/ffxiv_bossmod/pull/465/commits/80c7ea86e2ac96c624b2eff0f6a95d8fca689ce5))
image

### CODE REVIEW

File: BossMod/Autorotation/UIRotationWindow.cs [hunk 1]
```diff
@@ -90,13 +92,15 @@ public static bool DrawRotationSelector(RotationModuleManager mgr)
...
         {
-            if (ImGui.Button("X"))
+            if (ImGui.Button("Disable"))
             {
                 mgr.Preset = mgr.Preset == RotationModuleManager.ForceDisable ? null : RotationModuleManager.ForceDisable;
                 modified |= true;
             }
         }
 
+        ImGui.TextUnformatted("Presets:");
```
Comment (awgil) [inline BossMod/Autorotation/UIRotationWindow.cs:102]: Can we make it on same line as disable / X ? I'm somewhat conscious of vertical space taken by this window.

File: BossMod/Autorotation/UIRotationWindow.cs [hunk 2]
```diff
@@ -68,9 +65,14 @@ public override void Draw()
                 {
                     ImGui.SameLine();
                     using var style = ImRaii.PushColor(ImGuiCol.Text, 0xff00ffff);
-                    UIMisc.HelpMarker(() => "You have a preset activated, which fully overrides the plan!", FontAwesomeIcon.ExclamationTriangle);
+                    UIMisc.HelpMarker(() => "You have a preset activated, which fully overrides the CD plan!", FontAwesomeIcon.ExclamationTriangle);
                 }
             }
+
+            if (ImGui.Button("Open Boss Timeline"))
```
Comment (awgil) [inline BossMod/Autorotation/UIRotationWindow.cs:72]: I think i'll just remove this from here, and move to module viewer ui. It's really rarely useful.

File: BossMod/BossModule/BossModuleConfig.cs [hunk 3]
```diff
@@ -1,31 +1,51 @@
 ﻿namespace BossMod;
 
-[ConfigDisplay(Name = "Boss module window settings", Order = 1)]
+[ConfigDisplay(Name = "Boss Modules and Radar", Order = 1)]
```
Comment (awgil) [inline BossMod/BossModule/BossModuleConfig.cs:3]: You only moved stuff around, right? Not actually changed variable names? If you did, that would fuck up people's configs.

**Comment by Aimsucks** [comment]
image

Screenshot of updated window with requested changes.

**Pull request merged**